Client.java

```diff
@@ -1,15 +1,20 @@
 import java.net.*;
 import java.io.*;
+import java.security.*;
+import javax.crypto.*;
+import java.util.*;
 
 public class Client {
-      public static void startClient(String serverName, int serverPort) throws UnknownHostException, IOException {
+      public static void startClient(String serverName, int serverPort, char feature) throws UnknownHostException, IOException {
```

Comment:
`char feature` should be changed to a `Boolean integrity` (or is this confidentiality...)

Anyways, that way it will integrate nicely with the Security class.